bazel/external/quiche.BUILD

```diff
@@ -4708,7 +4708,7 @@ envoy_quiche_platform_impl_cc_library(
         "quiche/common/platform/default/quiche_platform_impl/quiche_googleurl_impl.h",
     ],
     deps = [
-        "@com_googlesource_googleurl//url",
+        "@com_github_google_gurl//url",
```

Comment:
what is the bcr name (if any)?

would be great if we can shorten this to just `@gurl`

Reply:
It will be googleurl : https://github.com/google/gurl/blob/main/MODULE.bazel